stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/internal/MessageInputFieldView.kt

```diff
@@ -188,7 +188,7 @@ internal class MessageInputFieldView : FrameLayout {
     }
 
     fun setCommandInputBadgeIcon(drawable: Drawable) {
```

Comment:
.

Reply:
I'm assuming a dot means "same here" haha

Reply:
Yeah, sorry! Just a continuation of what I was putting in! :]